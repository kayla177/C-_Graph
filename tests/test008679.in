ENTITY SWV1K7EY name_SWV1 author
ENTITY ONOP1IA0 name_ONOP author
ENTITY UWG16RFA name_UWG1 institution
ENTITY ZLONYWHL name_ZLON paper
ENTITY XODLPFR5 name_XODL author
ENTITY EEHWMDSH name_EEHW conference
ENTITY G2OHRE2P name_G2OH institution
ENTITY 64ITJ4YK name_64IT author
ENTITY 5VOVMEXZ name_5VOV author
ENTITY WE8K6JHB name_WE8K institution
RELATIONSHIP EEHWMDSH cited_by XODLPFR5 8.6
RELATIONSHIP EEHWMDSH collaborated_with WE8K6JHB 9.4
RELATIONSHIP UWG16RFA affiliated_with 64ITJ4YK 2.5
RELATIONSHIP 5VOVMEXZ presented_at UWG16RFA 9.1
RELATIONSHIP G2OHRE2P published_in 5VOVMEXZ 7.8
RELATIONSHIP EEHWMDSH presented_at ZLONYWHL 4.1
RELATIONSHIP G2OHRE2P affiliated_with ONOP1IA0 9.8
RELATIONSHIP G2OHRE2P published_in ZLONYWHL 7.8
RELATIONSHIP SWV1K7EY presented_at XODLPFR5 5.2
RELATIONSHIP UWG16RFA presented_at WE8K6JHB 6.4
RELATIONSHIP EEHWMDSH cited_by SWV1K7EY 5.5
RELATIONSHIP ZLONYWHL collaborated_with WE8K6JHB 6.5
RELATIONSHIP EEHWMDSH authored_by 5VOVMEXZ 8.5
RELATIONSHIP ONOP1IA0 affiliated_with SWV1K7EY 2.9
RELATIONSHIP UWG16RFA affiliated_with ZLONYWHL 10.0
PATH G2OHRE2P SWV1K7EY
HIGHEST
PRINT G2OHRE2P
PATH EEHWMDSH SWV1K7EY
PATH EEHWMDSH UWG16RFA
HIGHEST
PATH G2OHRE2P 5VOVMEXZ
PRINT WE8K6JHB
PRINT UWG16RFA
DELETE XODLPFR5
EXIT
