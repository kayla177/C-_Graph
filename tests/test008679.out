success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
G2OHRE2P 5VOVMEXZ EEHWMDSH SWV1K7EY 21.8
ONOP1IA0 XODLPFR5 34.7
5VOVMEXZ ONOP1IA0 ZLONYWHL
EEHWMDSH SWV1K7EY 5.5
EEHWMDSH 5VOVMEXZ UWG16RFA 17.6
ONOP1IA0 XODLPFR5 34.7
G2OHRE2P 5VOVMEXZ 7.8
EEHWMDSH UWG16RFA ZLONYWHL
64ITJ4YK 5VOVMEXZ WE8K6JHB ZLONYWHL
success
